src/corg/working-on-nuxeo-sources/compiling-nuxeo-from-sources.md

```diff
@@ -113,7 +113,7 @@ history:
 ## Requirements
 
 *   [Sun Java](http://java.sun.com/) 11 (since Nuxeo Platform 11.1), 8 (since Nuxeo Platform 7.2), 7 (since Nuxeo Platform 5.7) or 6 (up to Nuxeo Platform 5.6)
-*   [Apache Maven](http://maven.apache.org/) [2.2.1 up to Nuxeo 5.8](http://archive.apache.org/dist/maven/maven-2/2.2.1/binaries/) or [3.1.1+ since Nuxeo 5.9.2 (3.2+ recommended)](http://maven.apache.org/download.cgi)
+*   [Apache Maven](http://maven.apache.org/download.cgi) [3.3+](http://archive.apache.org/dist/maven/maven-3/) since Nuxeo 11.1, or [3.1.1+ (3.2+ recommended)](http://archive.apache.org/dist/maven/maven-3/) since Nuxeo 5.9.2 or [2.2.1](http://archive.apache.org/dist/maven/maven-2/2.2.1/binaries/) up to Nuxeo 5.8
```

Comment:
Same remark, don't we want to get rid of the reference to old platform versions?